src/main/webapp/jsp/instructorFeedbackResultsByGiverRecipientQuestion.jsp

```diff
@@ -470,9 +470,14 @@
                             <div class="col-md-10">
                     <%
                     	int qnIndx = 1;
+                        Boolean isQnCONTRIB = false; /* is the Question a Contribution Question? */
                         for (FeedbackResponseAttributes singleResponse : responsesFromGiverToRecipient.getValue()) {
                             FeedbackQuestionAttributes question = questions.get(singleResponse.feedbackQuestionId);
                             FeedbackQuestionDetails questionDetails = question.getQuestionDetails();
+                            isQnCONTRIB = false;
+                            if(questionDetails.questionType.getFeedbackQuestionDetailsInstance().getQuestionTypeDisplayName() == Const.FeedbackQuestionTypeNames.CONTRIB) {                             
```

Comment:
You can use questionDetails.questionType == FeedbackQuestionType.CONTRIB to compare the type directly, instead of comparing the display name.